service/src/main/java/com/softserve/borda/services/impl/UserServiceImpl.java

```diff
@@ -64,18 +64,14 @@ public void deleteUserById(Long id) {
     }
 
     @Override
-    public List<Board> getBoardsByUserId(Long id) {
-        User user = userRepository.findById(id).orElseThrow(
-                () -> new CustomEntityNotFoundException(User.class));
+    public List<Board> getBoardsByUser(User user) {
         return user.getUserBoardRelations().stream().map(
```

Comment:
Do not fetch entity-related entities using getters, use the service layer instead.

Reply:
Fixed in ReworkService branch/ticket